.idea/vcs.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="IssueNavigationConfiguration">
+    <option name="links">
+      <list>
+        <IssueNavigationLink>
+          <option name="issueRegexp" value="#(\d+)" />
+          <option name="linkRegexp" value="https://github.com/apache/dolphinscheduler/issues/$1" />
+        </IssueNavigationLink>
+      </list>
+    </option>
+  </component>
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="Git" />
+  </component>
+</project>
```

Comment:
Seems this will case conflicts If the dev users already have a `vcs.xml` file at local?

Reply:
The default configuration for this file is as follows:
```
<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="VcsDirectoryMappings">
    <mapping directory="" vcs="Git" />
  </component>
</project>
```
Most developers' configuration files should look like this.

I have now only added one component:
```
  <component name="IssueNavigationConfiguration">
    <option name="links">
      <list>
        <IssueNavigationLink>
          <option name="issueRegexp" value="#(\d+)" />
          <option name="linkRegexp" value="https://github.com/apache/dolphinscheduler/issues/$1" />
        </IssueNavigationLink>
      </list>
    </option>
  </component>
```
I am not sure if other developers will modify this file to implement other functionalities.

With IDEA's issue tracker, we can effortlessly navigate to Github issues or pull requests, facilitating a better understanding of the change history for developers.

Reply:
Ok.